docs/release-process.md

```diff
@@ -183,6 +185,7 @@ k8s-container-image-promoter --thin-manifest-dir k8s.gcr.io
 Currently we send the image and non-image artifact promotion PRs separately.
 
 ```
+cd ${GOPATH}/src/k8s.io/k8s.io
```

Comment:
```suggestion
cd ${GOPATH}/src/k8s.io/k8s.io
git checkout main
git pull
```

Reply:
Already done in "Create container promotion PR" step.

Reply:
But they why do you add a `cd`? It would make sense to either assume the same context of the previous step or to reestablish it from scratch, but not to reestablish only part of the context.

Reply:
Makes sense when you skip previous step.

Reply:
But if you skip the previous step you might not be on the head of `master`.

Reply:
Previous step as in "You can dry-run the promotion...".